src/transform/tests/test_runner.rs

```diff
@@ -137,8 +136,8 @@ mod tests {
         let format_type = get_format_type(args);
 
         let out = match test_type {
-            TestType::Opt => {
-                for transform in FULL_TRANSFORM_LIST.iter() {
+            TestType::Opt => FULL_TRANSFORM_LIST.with(|transforms| -> Result<_, Error> {
```

Comment:
As a side note, where do I find the documentation for this `with` method?

Reply:
Here https://doc.rust-lang.org/std/thread/struct.LocalKey.html#method.with